data/telegraf_plugins.yml

```diff
@@ -141,6 +141,15 @@ input:
     introduced: 1.12.0
     tags: [linux, macos, windows, systems]
 
+  - name: Arista LANZ Consumer
+    id: lanz
+    description: |
+      The Arista LANZ Consumer input plugin provides a consumer for use with Arista Networks’ Latency Analyzer (LANZ).
+      Metrics are read from a stream of data via TCP through port 50001 on the switches management IP. Data is in Protobuffers format.
```

Comment:
Should "switches" be "switch's"

Reply:
thanks, @sanderson, @danielnelson confirmed